assets/js/googlesitekit-admin.js

```diff
@@ -28,7 +28,15 @@ if ( 'toplevel_page_googlesitekit-dashboard' !== window.pagenow && 'site-kit_pag
 	appendNotificationsCount( count );
 }
 
-const wpLogout = document.querySelector( '#wp-admin-bar-logout a' );
-wpLogout.addEventListener( 'click', () => {
-	clearAppLocalStorage();
-} );
+let wpLogout = document.querySelector( '#wp-admin-bar-logout a' );
```

Comment:
@lucymtc 
can you test it with this way:
```
const wpLogout = document.querySelector( '#wp-admin-bar-logout a' ) || document.querySelector( '.sidebar__me-signout button' );
```
and see if it is working?
If yes and i think it's simpler.

Reply:
I think the way it's written right now is more readable, so I'd prefer to leave it as is.